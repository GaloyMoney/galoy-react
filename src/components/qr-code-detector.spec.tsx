import { render } from "@testing-library/react"
import { QRCodeDetecor } from "./qr-code-detector"

describe("SatSymbol", () => {
  it("renders and matches snapshot", () => {
    const { asFragment } = render(
      <QRCodeDetecor
        startText="start"
        stopText="stop"
        onCodeDetected={jest.fn()}
        onValidCode={jest.fn()}
      />,
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
