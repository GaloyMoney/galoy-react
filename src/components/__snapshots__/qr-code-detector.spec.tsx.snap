// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SatSymbol renders and matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="scan link center-display"
    >
      start
    </div>
  </div>
</DocumentFragment>
`;
